drivers/clock_control/nrf_clock_calibration.c

```diff
@@ -300,10 +294,7 @@ void z_nrf_clock_calibration_lfclk_stopped(void)
 
 void z_nrf_clock_calibration_isr(void)
```

Comment:
please rename this function since isr might be misleading. Something like `z_nrf_clock_calibration_done()`?